ipaclient/install/ipa_epn.py

```diff
@@ -551,6 +552,10 @@ def _send_emails(self):
                 mail_from = api.env.mail_from
             else:
                 mail_from = "noreply@%s" % self.default_email_domain
+            if api.env.mail_from_realname:
+                mail_from_realname = api.env.mail_from_realname
+            else:
+                mail_from_realname = "IPA-EPN"
```

Comment:
You don't need this logic. On like 67 setting the default to "IPA-EPN" is sufficient, then use api.env.mail_from_realname in the call to _send_emails()